src/Sulu/Component/Content/Metadata/Loader/XmlLegacyLoader.php

```diff
@@ -72,6 +72,8 @@ class XmlLegacyLoader implements LoaderInterface
         'shadow-base',
         'author',
         'authored',
+        'type',
+        'id'
```

Comment:
We also put a comma after the last array item :slightly_smiling_face: That will reduce that diff size if we add more items later.